### PR TITLE
(PE-12968) Add node config and scenario configs for PE 2015.2.3

### PR DESCRIPTION
**Pull request by haus**
This commit adds a node config and some scenarios for the PE 2015.2.3
recording.

### CODE REVIEW

**Comment by KevinCorcoran** [comment]
:see_no_evil:

File: simulation-runner/config/nodes/pe201523-catalogzero.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "certname": "w3h244u9nwii0y5.delivery.puppetlabs.net",
+    "simulation_class": "com.puppetlabs.gatling.node_simulations.PE201523CatalogZero",
```
Comment (nwolfe) [inline simulation-runner/config/nodes/pe201523-catalogzero.json]: I don't see this class in this PR - should this be one of the `WithFeeder` classes?

Comment (haus) [inline simulation-runner/config/nodes/pe201523-catalogzero.json]: Don't worry, already merged. It's here: https://github.com/puppetlabs/gatling-puppet-load-test/blob/master/simulation-runner/src/main/scala/com/puppetlabs/gatling/node_simulations/PE201523CatalogZero.scala

**Comment by nwolfe** [comment]
:+1:

**Pull request merged**